api/src/main/java/org/openmrs/module/idgen/service/IdentifierSourceService.java

```diff
@@ -265,6 +265,13 @@
      */
     @Transactional(readOnly=true)
     IdentifierSource getIdentifierSourceByUuid(String uuid);
+    
+    /**
+     * @param patientIdentifierType

```

Comment:
Can you put a description for this method and the above parameter?

Reply:
description added